setup.py

```diff
@@ -551,6 +554,7 @@ def do_setup():
             'grpc': grpc,
             'hdfs': hdfs,
             'hive': hive,
+            'hvac': hvac,
```

Comment:
This name isn't very descriptive if you don't already know what the module acronym does (and I can't see any docs) about extras? How about calling this extra `hasicorp`?

Reply:
Updated